src/structures/VoiceChannel.js

```diff
@@ -0,0 +1,14 @@
+const Base = require('./Base');
+
+module.exports = class VoiceChannel extends Base {
+  constructor(data) {
+    super(data.id)
+
+     this.type = data.type
```

Comment:
The project's indentation pattern is 4 spaces so we remove one space
```suggestion
    this.type = data.type
```